README.md

```diff
@@ -62,11 +62,11 @@ The success of our vision to bring knowledge offline depends on you. Even a smal
 
 Currently, we have over **700 contributors**
 
-When you contribute, your name with a link (if available) is added to our [contributors list](https://github.com/OpenGenus/cosmos/wiki/contributors).
+When you contribute, your name with a link (if available) is added to our [contributor's list](https://github.com/OpenGenus/cosmos/wiki/contributors).
 
-You can contribute by writing `code`, documentation in the form of `installation guides` and `style guides`, making Cosmos search friendly and many others. There are endless possibilities.
+You can contribute by writing `code`, documentation in the form of `installation guides`, and `style guides`, making Cosmos search friendly and many others. There are endless possibilities.
 
-Additionally, you might want to take a look at this [contributing guidelines](https://github.com/OpenGenus/cosmos/wiki/contribute) before you make Cosmos better.
+Additionally, you might want to take a look at these [contributing guidelines](https://github.com/OpenGenus/cosmos/wiki/contribute) before you make Cosmos better.
 
 You may, also, refer to the available [style guides](/guides/coding_style) before contributing code.
```

Comment:
You may also, refer to thee available [style guides](/guides/coding_style) before contributing code.